CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 88 Release Notes
 
+### New table 'GROUP_METRICS_ENTRY'
```

Comment:
Late-breaking comment I noticed while reorganizing the release notes... now that we have a decent amount of them, maybe we should group this and the index removal under a `### Database` section.

Reply:
Yeah I can do that 👍